Highbar.Functions.Test/FunctionsTest.cs

```diff
@@ -8,6 +8,7 @@ namespace Highbar.Functions.Test
 {
   public class FunctionsTests
   {
+    private static readonly string _testString = "string";
```

Comment:
Consider the non-`Validation` additions that are not required for `Validation` be completed in a separate request.